packages/block-editor/src/components/child-layout-control/index.js

```diff
@@ -60,9 +67,19 @@ export default function ChildLayoutControl( {
 		type: parentType,
 		default: { type: defaultParentType = 'default' } = {},
 		orientation = 'horizontal',
+		columnCount,
 	} = parentLayout ?? {};
 	const parentLayoutType = parentType || defaultParentType;
-
+	const gridColumnNumber = parseInt( columnCount, 10 ) || 3;
+	const rootClientId = useSelect( ( select ) =>
+		select( blockEditorStore ).getBlockRootClientId( panelId )
+	);
+	const { moveBlocksToPosition } = useDispatch( blockEditorStore );
+	const getBlocksBeforeCurrentCell = useGetBlocksBeforeCurrentCell(
```

Comment:
Welp, had to add a bunch of hooks here to be able to move the blocks in the markup order 😅 
Not sure if there's a better way of doing this. A possible alternative would be not exposing these controls in the sidebar at all (which would mean keyboard manipulation would depend exclusively on the block movers, maybe not ideal)